### PR TITLE
feat: migrations

### PR DESCRIPTION
**Pull request by Arash217**

### CODE REVIEW

File: scripts/migrations/index.mjs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  }, ({ name }) => {
+    const { stdout, stderr } = spawnSync('npx', [
+      'datocms',
+      'migrations:new',
+      `'${ name }'`,
+      `--api-token=${ apiTokens[0] }`,
+    ])
+
+    console.log(stderr.toString())
+    console.log(stdout.toString())
```
Comment (sjoerdbeentjes) [inline scripts/migrations/index.mjs:38]: This can be removed right?

File: scripts/migrations/index.mjs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { hideBin } from 'yargs/helpers'
+
+import { config } from 'dotenv-safe'
+config()
+
+import { spawnSync } from 'child_process'
+
+const {
+  DATO_FULL_ACCESS_API_TOKEN_RWS_VIEWER,
+  DATO_FULL_ACCESS_API_TOKEN_DELTARES_VIEWER,
```
Comment (sjoerdbeentjes) [inline scripts/migrations/index.mjs:13]: Let's rename this to `DATO_FULL_ACCESS_API_TOKEN_OPENEARTH_VIEWER`

File: scripts/migrations/index.mjs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+} = process.env
+
+const apiTokens = [
+  DATO_FULL_ACCESS_API_TOKEN_DELTARES_VIEWER,
+  DATO_FULL_ACCESS_API_TOKEN_RWS_VIEWER,
+]
```
Comment (sjoerdbeentjes) [inline scripts/migrations/index.mjs:19]: Nice solution!

**Review by sjoerdbeentjes — CHANGES_REQUESTED** [review]
Looking good! Added some small remarks

File: scripts/migrations/index.mjs (hunk 1 above)
Comment (Arash217) [reply scripts/migrations/index.mjs:38]: No, that logs the output from DatoCMS CLI back to the user.

File: scripts/migrations/index.mjs (hunk 2 above)
Comment (Arash217) [reply scripts/migrations/index.mjs:13]: Which one should be renamed, `DATO_FULL_ACCESS_API_TOKEN_RWS_VIEWER` or `DATO_FULL_ACCESS_API_TOKEN_DELTARES_VIEWER`?

**Pull request closed without merging**